lib/usd/ui/views/USDImportDialog.cpp

```diff
@@ -36,14 +37,29 @@ USDImportDialog::USDImportDialog(const std::string& filename, QWidget* parent)
 {
 	if (!fStage)
 		throw std::invalid_argument("Invalid filename passed to USD Import Dialog");
+	if (mayaQtUtil == nullptr)
+		throw std::invalid_argument("Invalid IMayaMQtUtil passed to USD Import Dialog");
```

Comment:
@kxl-adsk Do we have anything in the coding guidelines on using exceptions (or **not** using them)?

I'm not sure what, if anything, catches these?

Reply:
We have a mention in C++ core guidelines which talks about exceptions and alternative solution. https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#i10-use-exceptions-to-signal-a-failure-to-perform-a-required-task

Reply:
An additional guideline would be about noexcept  https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-noexcept